processor/src/main/java/com/umairjavid/kombind/processor/AdapterProcessor.kt

```diff
@@ -0,0 +1,83 @@
+package com.umairjavid.kombind.processor
+
+import com.google.auto.service.AutoService
+import com.squareup.kotlinpoet.ClassName
+import com.squareup.kotlinpoet.FileSpec
+import com.squareup.kotlinpoet.FunSpec
+import com.squareup.kotlinpoet.KModifier
+import com.squareup.kotlinpoet.ParameterSpec
+import com.squareup.kotlinpoet.ParameterizedTypeName.Companion.parameterizedBy
+import com.squareup.kotlinpoet.PropertySpec
+import com.squareup.kotlinpoet.TypeName
+import com.squareup.kotlinpoet.TypeSpec
+import com.squareup.kotlinpoet.asTypeName
+import com.umairjavid.kombind.anontation.SimpleKombindAdapter
+import java.io.File
+import javax.annotation.processing.AbstractProcessor
+import javax.annotation.processing.Processor
+import javax.annotation.processing.RoundEnvironment
+import javax.lang.model.SourceVersion
+import javax.lang.model.element.TypeElement
+import javax.lang.model.element.VariableElement
+
+
+@AutoService(Processor::class)
+class AdapterProcessor : AbstractProcessor() {
+    override fun getSupportedAnnotationTypes() = mutableSetOf(SimpleKombindAdapter::class.java.name)
+
+    override fun getSupportedSourceVersion() = SourceVersion.latestSupported()
+
+    override fun process(typeElemnts: MutableSet<out TypeElement>?, roundEnv: RoundEnvironment?): Boolean {
+        val env = roundEnv!!.getElementsAnnotatedWith(SimpleKombindAdapter::class.java)
+        env.forEach {
+            val layoutRes = it.getAnnotation(SimpleKombindAdapter::class.java).layoutRes
+            if (it is VariableElement)
+                generateAdapterClass(it, layoutRes)
+        }
+        return false
```

Comment:
We always return false?

Reply:
From my understanding , this method must always return false, if not other annotations/rounds will not be processed.